kalite/distributed/management/commands/restorebackup.py

```diff
@@ -0,0 +1,42 @@
+from os.path import join, expanduser
+import os
+from django.core.management.base import BaseCommand
+from django.core.management import call_command
+from optparse import make_option
+BACKUP_DIRPATH = expanduser("~")
+BACKUP_DIRPATH = os.path.join(BACKUP_DIRPATH, 'ka-lite-backups')
+
+backup_filepath=""
```

Comment:
To clarify -- we want to avoid global variables. Only use them if they're a constant.